indy_node/utils/node_control_tool.py

```diff
@@ -271,7 +271,8 @@ def _run_shell_script(cls, command, timeout):
                               timeout=timeout)
 
     def start(self):
-        self._hold_packages()
+        if shutil.which("apt-mark"):
+            self._hold_packages()
```

Comment:
It will work. But I don't think that it is to skip hold-packages stage silently. Also if we someday want to call `_hold_packages` from some other place we should copy-paste the conditional you added. Does it make sense to move the conditional inside the method itself? Could you add some logging/warning in case no operations performed?

Reply:
Please, add logging (warning) for the case when we skip packages holding. Thank you